print/server-middleware/routes/pdfGeneratorNativeChrome.js

```diff
@@ -33,6 +33,14 @@ function measurePerformance(msg) {
   console.log(msg || '')
 }
 
+if (!process.sentry) {
+  process.sentry = {
```

Comment:
Hmm, not sure I like this way of overriding `process.sentry`. What if some other code somewhere checks for `process.sentry` at one point in the future, and then gets confused, because `process.sentry` it's not an actual sentry object.

If you don't want to have the if-clause below, wouldn't it be cleaner to have a `captureException` method here which is responsible to either raise a `sentry.captureException` or simply log to the console, depending on whether sentry is available or not?